string plugin_name

---
bool result